### DISCUSSION TITLE
(1) Disable notification count in page title

### DISCUSSION THREAD

**Discussion by quenz** [typefully/minimal-twitter#101] [Ideas]
Can be very distracting for me. Makes me want to go to my notifications just to get rid of it, when I only came to twitter to look at a single tweet. 99% of my notifications are the "in case you missed" ones too, which can't be disabled.

A toggle to hide the notification count on the notifications button might be nice too, but the one in the title is more important. At least if I can disable that, twitter can't try to draw me in simply by having a twitter tab open in the background.

**Comment by ghost** [comment]
@noopnoop started a PR (https://github.com/thomaswang/minimal-twitter/pull/89) for this, will review for next minor release!

**Reply by kannapoix** [reply]
@thomaswang 
Thank you for maintaining this extension.

I also want this feature!
Any progress?
